docs/tables/crowdstrike_zta_assessment.md

```diff
@@ -0,0 +1,46 @@
+# Table: crowdstrike_zta_assessment
+
+Zero Trust Assessment is a Falcon Insight feature that monitors the native OS security settings and applied sensor policies of Windows 10 and macOS endpoints to gauge the device security posture of managed assets within an organization. The metrics derived from these assessments can be used by IdP, NAC, and other solutions to provide additional data points around device health and security posture as part of their respective conditional access capabilities.
+
+## Examples
```

Comment:
What type of info is included in the `assessment_items` column? Are there examples we can create using that column?

Reply:
See last example. This is taken from Ciaran's suggestion